src/aap_eda/settings/default.py

```diff
@@ -346,6 +346,45 @@ def _get_databases_settings() -> dict:
 REDIS_CLIENT_KEY_PATH = settings.get("MQ_CLIENT_KEY_PATH", None)
 REDIS_DB = settings.get("MQ_DB", 0)
 
+
+def _rq_common_parameters():
+    params = {
+        "DB": REDIS_DB,
+        "USERNAME": REDIS_USER,
+        "PASSWORD": REDIS_USER_PASSWORD,
+    }
+    if REDIS_UNIX_SOCKET_PATH:
+        params["UNIX_SOCKET_PATH"] = REDIS_UNIX_SOCKET_PATH
+    else:
+        params |= {
+            "HOST": REDIS_HOST,
+            "PORT": REDIS_PORT,
+        }
+        if REDIS_CLIENT_CERT_PATH:
+            params["SSL"] = True
+    return params
+
+
+def _rq_redis_client_parameters():
+    params = {}
+    if (not REDIS_UNIX_SOCKET_PATH) and REDIS_CLIENT_CERT_PATH:
+        params |= {
+            "ssl_certfile": REDIS_CLIENT_CERT_PATH,
+            "ssl_keyfile": REDIS_CLIENT_KEY_PATH,
+            "ssl_ca_certs": REDIS_CLIENT_CACERT_PATH,
+        }
+    return params
+
+
+def rq_test_redis_client_parameters():
```

Comment:
This function is for tests, should not be here. Is it a mistake or intended?

Reply:
It is intended.  It eliminates duplication of logic to establish the correct group of parameters to connect to redis for both non-TLS and TLS.

If we want to locate it in some utility source that's fine and will also mean moving the other functions on which it depends to avoid any code "reaching" into settings (something I didn't want to do with tests, but figured it might be ok for that specific use).

Reply:
OK.  That makes sense.